<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dog</class>
 <widget class="QWidget" name="Dog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QToolButton" name="toolButton_home">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Adobe Gothic Std</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>PetFinder</string>
         </property>
         <property name="icon">
          <iconset resource="../project/Resources.qrc">
           <normaloff>:/imgs/CommunityIcons_home.png</normaloff>:/imgs/CommunityIcons_home.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Please choose a state</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBoxStates">
               <property name="editable">
                <bool>true</bool>
               </property>
               <item>
                <property name="text">
                 <string/>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Please choose a city</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBoxCities">
               <property name="editable">
                <bool>true</bool>
               </property>
               <item>
                <property name="text">
                 <string/>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QComboBox" name="combobox_Breeds">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>Breeds</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>American Curl</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="combobox_Age">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>Age</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>puppy</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>adult</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_Gender">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>Gender</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>female</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>male</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_Size">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>Size</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>small</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>mid</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>large</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_Color">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>Color</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>black</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>brown</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>grey</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>white</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>mixed</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>spotted</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>stripped</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_CoatLength">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>Coat length</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>no hair</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>short hair</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>mid-length hair</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>long hair</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_activeLevel">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>Active level</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>low</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>mid</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>high</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_SearchText">
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Enter Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_search">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_DogLanguage">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>Language</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>English</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Spanish</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_SortBy">
             <item>
              <property name="text">
               <string>Sort By:</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Best Match</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Distance (from lowest to highest)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Distance (from highest to lowest)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="minimumSize">
          <size>
           <width>550</width>
           <height>0</height>
          </size>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>534</width>
            <height>1000</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>1000</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../project/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
